feature/aks-cluster-creation/azure-subnet.tf

```diff
@@ -0,0 +1,20 @@
+#####################################################
+# DO NOT Modify by hand - Managed by Automation
+#####################################################
+#####################################################
+# This file can be used as a base template to build other Terraform files. It attempts to use as much
+# Terraform interprolation as possible by creating Terraform variables instead of changing inline
+# this approach provides an easier way to do creative looping, fetch IDs of created resources etc.
+#####################################################
+#####################################################
+# {{ specification.name }}
+#####################################################
```

Comment:
how is that managed by automation?

Reply:
Removed, historical lines, I don't know why but I didn't focused on these lines/comments.